sdk/extendedlocation/ci.yml

```diff
@@ -0,0 +1,39 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - hotfix/*
+      - release/*
+  paths:
```

Comment:
The exclude for `sdk/extendedlocation/pom.xml` doesn't make sense given the include is only for `ci.yml`

Comparing this to [sdk/healthbot/ci.yml](https://github.com/azure-sdk/azure-sdk-for-java/blob/master/sdk/healthbot/ci.yml), it looks like either the include is too restrictive, or the `/pom.xml` exclude could be dropped.

Reply:
In this case, we implicitly exclude `sdk/extendedlocation/pom.xml` because it's not part of the includes.

It's not a problem, I was just pointing out that we were explicitly excluding a thing that isn't included.

Reply:
Oh I see. Yes `sdk/extendedlocation/pom.xml` actually not included at all :-)